gateway/gateway.go

```diff
@@ -1266,6 +1280,7 @@ func (gateway *HandleT) StartWebHandler(ctx context.Context) error {
 	srvMux.HandleFunc("/", WithContentType("application/json; charset=utf-8", app.LivenessHandler(gateway.jobsDB))).Methods("GET")
 	srvMux.HandleFunc("/v1/import", gateway.webImportHandler).Methods("POST")
 	srvMux.HandleFunc("/v1/audiencelist", gateway.webAudienceListHandler).Methods("POST")
+	srvMux.HandleFunc("/v1/extract", gateway.webExtractHandler).Methods("POST")
```

Comment:
Can we do `/internal/v1/extract` instead? This way we can easily block unwanted access and make sure that this is used internally only (i.e. by making sure that everything that starts with `/internal` isn't accessible from outside).

Reply:
Addressed